libraries/botbuilder-azure/src/azureBlobTranscriptStore.ts

```diff
@@ -307,14 +386,15 @@ export class AzureBlobTranscriptStore implements TranscriptStore {
             getBlobToTextAsync: this.denodeify(blobService, blobService.getBlobToText),
             deleteBlobIfExistsAsync: this.denodeify(blobService, blobService.deleteBlobIfExists),
             listBlobsSegmentedWithPrefixAsync: this.denodeify(blobService, blobService.listBlobsSegmentedWithPrefix),
+            // tslint:disable-next-line:no-string-literal
```

Comment:
This one I included because `listBlobDirectoriesSegmentedWithPrefix` doesn't seem to be a declared property on blobService.  I wasn't sure how to remedy that -- there is a subclass defined at the bottom of that file, but it looks to be defining the async versions of existing functions. Any ideas?

Reply:
I'll file a cleanup issue for this; searching for the method name, `listBlobDirectoriesSegmentedWithPrefix` in Azure/azure-storage-node revealed a couple of issues and PRs related to adding the method to the d.ts files. 

Issue: Azure/azure-storage-node#504
PR: Azure/azure-storage-node#506

Reply:
:shipit: